asteroid/masknn/recurrent.py

```diff
@@ -484,3 +493,81 @@ def get_config(self):
             "bidirectional": self.bidirectional,
         }
         return config
+
+
+class DCCRMaskNetRNN(nn.Module):
+    """RNN (LSTM) layer between encoders and decoders introduced in [1].
+
+    Args:
+        in_size (int): Number of inputs to the RNN. Must be the product of non-batch,
+            non-time dimensions of output shape of last encoder, i.e. if the last
+            encoder output shape is [batch, n_chans, n_freqs, time], `in_size` must be
+            `n_chans * n_freqs`.
+        hid_size (int, optional): Number of units in RNN.
+        rnn_type (str, optional): Type of RNN to use. See ``SingleRNN`` for valid values.
+        norm_type (Optional[str], optional): Norm to use after linear.
+            See ``asteroid.masknn.norms`` for valid values. (Not used in [1]).
+
+    References:
+        [1] : "DCCRN: Deep Complex Convolution Recurrent Network for Phase-Aware Speech Enhancement",
+        Yanxin Hu et al.
+        https://arxiv.org/abs/2008.00264
+    """
+
+    def __init__(self, in_size, hid_size=128, rnn_type="LSTM", norm_type=None):
+        super().__init__()
+
+        self.rnn = ComplexSingleRNN(rnn_type, in_size, hid_size)
+        self.linear = complex_nn.ComplexMultiplicationWrapper(nn.Linear, hid_size, in_size)
```

Comment:
Why the RNN as `ComplexSingleRNN` and the linear layer not? Just wondering.

Reply:
Good point, changed it.